package.json

```diff
@@ -4,7 +4,9 @@
   "description": "A parser for the SPARQL query language",
   "author": "Ruben Verborgh <ruben.verborgh@gmail.com>",
   "dependencies": {
-    "n3": "^1.0.4"
+    "expect": "^24.8.0",
```

Comment:
Is this needed? Chai should be sufficient I think.

Reply:
This library makes it easier to add your own matcher functions, and since we only used Chai for the expect functionality I thought it made sense to import that functionality seperately.

Reply:
I'll let @RubenVerborgh decide on this one :-)

Reply:
I suggest to move this to `devDependencies` as it's only for tests, right?